lite/configure.ac

```diff
@@ -177,12 +177,6 @@ XMP_TRY_COMPILE(whether compiler understands -Wunknown-warning-option,
   int main(void){return 0;}],
   CFLAGS="${CFLAGS} -Wno-unknown-warning-option")
 
-XMP_TRY_COMPILE(whether compiler understands -Wunused-but-set-variable,
-  ac_cv_c_flag_w_unused_but_set_variable,
-  -Wunused-but-set-variable,[
-  int main(void){return 0;}],
-  CFLAGS="${CFLAGS} -Wno-unused-but-set-variable")
-
 XMP_TRY_COMPILE(whether compiler understands -Wunused-result,
   ac_cv_c_flag_w_unused_result,
```

Comment:
Why not remove `-Wunused-result` and `-Warray-bounds` from lite version too?

Reply:
Good catch, I forgot that...

Reply:
FYI I vaguely remember a point in the recent past when glibc was very aggressive about `__attribute__((warn_unused_result))` usage on stdio, which might have been why that was added. I don't know if that was changed at some point.